README.md

```diff
@@ -1,2 +1,96 @@
-# merge-sdk-kotlin
-The Kotlin/JVM SDK for accessing various Merge Unified APIs
+# Merge-SDK-Kotlin
+
+The Kotlin/JVM SDK for accessing various Merge Unified APIs. We use the following dependencies:
+
+* Ktor for http communication
+  * OkHttp engine (labeled Apache in ktor)
+  * Jackson serialization engine
+    * JSR 310 (java.time.OffsetDateTime for datetimes)
+* JUnit for tests
+* NO logging, aside from some `println` in tests
+
+We target Java language level 8 and above. The API methods are written using Kotlin coroutines, but can be called with
+`CompletableFutures` see [https://stackoverflow.com/a/52887677](this example). There should also be some packages that
```

Comment:
could you provide a Java test file, or a more detailed section in the readme on how to accomplish this?